release-notes/1.0/1.0-supported-os.md

```diff
@@ -30,10 +30,10 @@ Mac OS X                      | 10.11, 10.12*                  | x64          |
 OS                            | Version                        | Architectures| Notes
 ------------------------------|--------------------------------|--------------|-----
 Red Hat Enterprise Linux <br> CentOS <br> Oracle Linux     | 7                             | x64            | [Red Hat support policy](https://access.redhat.com/support/policy/updates/errata/) <br> [CentOS lifecycle](https://wiki.centos.org/FAQ/General#head-fe8a0be91ee3e7dea812e8694491e1dde5b75e6d) <br> [Oracle Linux lifecycle](http://www.oracle.com/us/support/library/elsp-lifetime-069338.pdf)
-Fedora                        | 24                             | x64          | [Fedora lifecycle](https://fedoraproject.org/wiki/End_of_life)
+Fedora                        | 26                             | x64          | [Fedora lifecycle](https://fedoraproject.org/wiki/End_of_life)
 Debian                        | 8.2+                           | x64          | [Debian lifecycle](https://wiki.debian.org/DebianReleases)
-Ubuntu <br> Linux Mint        | 14.04, 16.04, 16.10* <br> 17      | x64            | [Ubuntu lifecycle](https://wiki.ubuntu.com/Releases) <br> [Linux Mint end of life announcements](https://forums.linuxmint.com/search.php?keywords=%22end+of+life%22&terms=all&author=&sc=1&sf=titleonly&sr=posts&sk=t&sd=d&st=0&ch=300&t=0&submit=Search)
-openSUSE                      | 42.1+ (1.1)                           | x64          | [OpenSUSE lifecycle](https://en.opensuse.org/Lifetime)
+Ubuntu <br> Linux Mint        | 14.04, 16.04 <br> 17      | x64            | [Ubuntu lifecycle](https://wiki.ubuntu.com/Releases) <br> [Linux Mint end of life announcements](https://forums.linuxmint.com/search.php?keywords=%22end+of+life%22&terms=all&author=&sc=1&sf=titleonly&sr=posts&sk=t&sd=d&st=0&ch=300&t=0&submit=Search)
+openSUSE                      | 42.3+ (1.1)                           | x64          | [OpenSUSE lifecycle](https://en.opensuse.org/Lifetime)
```

Comment:
What about 1.0 which currently just has a package for OpenSuse 13.2? Do we want to ship a new package for 1.0 as well, or do we only care about updating the support map for 1.1?

Reply:
only care